python/src/IntegrationExpansion_doc.i.in

```diff
@@ -0,0 +1,57 @@
+%feature("docstring") OT::IntegrationExpansion
+"L2 approximation on an orthonormal basis using least-squares and a fixed basis.
+
+Available constructors:
+    IntegrationExpansion(*inputSample, outputSample, distribution*)
+
+    IntegrationExpansion(*inputSample, weights, outputSample, distribution*)
+
+    IntegrationExpansion(*inputSample, outputSample, distribution, basis, basisSize*)
```

Comment:
In order to be able to directly set the `indices`, we can provide the following constructor: 
```suggestion
    IntegrationExpansion(*inputSample, outputSample, distribution, basis, basisSize*)

    IntegrationExpansion(*inputSample, outputSample, distribution, basis, indices*)
```
where `indices` is an instance of the `Indices` class.

Reply:
I think that you miss the point. You can easily integrate this suggestion using online [github features](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request).

Reply:
The spirit of these new classes is to mimic the existing capabilities with LeastSquareStrategy+FixedStrategy. The ability to select a given set of functions has been implemented the same way it has been implemented in the DesignProxy class:
 * you start with all the functions up to a given basis size
 * you select which functions are active at a given point
It allows to reuse efficiently the DesignProxy when the selection changes. 

BTW if I integrate your suggestion I get only one additional line in the documentation or also the C++ code and the associated tests? Adding a new constructor is not so innocuous...